src/OmnichannelChatSDK.ts

```diff
@@ -1109,15 +1549,21 @@ class OmnichannelChatSDK {
         }
     }
 
-    private async getChatConfig(): Promise<IChatConfig> {
+    private async getChatConfig(): Promise<ChatConfig> {
         try {
             const liveChatConfig = await this.OCClient.getChatConfig();
             const {
                 DataMaskingInfo: dataMaskingConfig,
                 LiveChatConfigAuthSettings: authSettings,
-                LiveWSAndLiveChatEngJoin: liveWSAndLiveChatEngJoin
+                LiveWSAndLiveChatEngJoin: liveWSAndLiveChatEngJoin,
+                LiveChatVersion: liveChatVersion
             } = liveChatConfig;
 
+            this.liveChatVersion = liveChatVersion || LiveChatVersion.V1;
```

Comment:
how is liveChatVersion different from this.liveChatVersion? Where do we set the value of liveChatVersion?

Reply:
`this.liveChatVersion` is the private attribute of `OmnichannelChatSDK`. `liveChatVersion` is the live chat version picked from getting the chat config. This line states to save the live chat version from chat config, or set it to `LiveChatVersion.V1` if the value is not undefined from the chat config